tests/test_framework/phpbb_functional_test_case.php

```diff
@@ -1496,10 +1496,10 @@ protected function set_flood_interval($flood_interval)
 	}
 
 	/**
-	* Check if a user exists by username(s) or user_id(s)
+	* Check if a user exists by username or user_id
 	*
-	* @param array &$user_id_ary The user ids to check or empty if usernames used
-	* @param array &$username_ary The usernames to check or empty if user ids used
+	* @param string $username The username to check or empty if user_id used
+	* @param int $user_id The use id to check or empty if username used
```

Comment:
`The user id to check or empty if username used` ?

Reply:
id instead of identifier is the way to go, should say user and not "use" though :)